.travis.yml

```diff
@@ -27,17 +27,17 @@ matrix:
       services: docker
       env: DOCKER_IMAGE=ubuntu:16.04
     - os: osx
-      osx_image: xcode9.2
+      osx_image: xcode9.3
       sudo: required
       env: JAZZY_ELIGIBLE=true
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=swift-4.1-DEVELOPMENT-SNAPSHOT-2018-03-04-a
+      env: SWIFT_SNAPSHOT=4.0.3
     - os: osx
       osx_image: xcode9.2
       sudo: required
-      env: SWIFT_SNAPSHOT=swift-4.1-DEVELOPMENT-SNAPSHOT-2018-03-04-a
+      env: SWIFT_SNAPSHOT=4.0.3
```

Comment:
@djones6 do you know whether this line is needed?  Is it enough just to have `osx_image: xcode9.2`?